do/droplets.go

```diff
@@ -150,25 +156,60 @@ func (i *instances) dropletById(ctx context.Context, id string) (*godo.Droplet,
 	return droplet, nil
 }
 
+func (i *instances) dropletList(ctx context.Context) ([]godo.Droplet, error) {
```

Comment:
dropletList is also `godo` function name.
Could `allDropletList` do a better fit?